guide/english/computer-hardware/gpu/index.md

```diff
@@ -53,19 +53,19 @@ Just recently, Nvidia launched a new line of GPUs titled RTX which includes the
 
 ## Basic Unified GPU Architecture Components
 
-Unified GPU architectures are based on a parallel array of many programmable processors, wherein all the stages of graphics pipeline, viz., vertex, geometry, rasterization, and pixel shader processing and parallel computations on the same core, in contrast with earlier GPUs. The processor array is highly integrated with fixed function processors for compression and decompression, rasterization, raster operations, texture filtering, anti-aliasing, video decoding, and HD video processing.
+Unified GPU architectures are based on a parallel array of many programmable processors, wherein all the stages of the graphics pipeline, viz., vertex, geometry, rasterization, and pixel shader processing and parallel computations on the same core, in contrast with earlier GPUs. The processor array is highly integrated with fixed function processors for compression and decompression, rasterization, raster operations, texture filtering, anti-aliasing, video decoding, and HD video processing.
 
 The following discussed architecture is focused on executing many parallel threads efficiently on many processor cores.
 
 ### Processor Array
-A processor array consists of many processing cores. A unified GPU processor array has a typical organized structure of multi-threaded multi-processors. For execution of each thread, a multiprocessor is involved, and in each GPUs multi-processor, also known as Streaming Multiprocessors (SM), there are numerous Streaming processors, arranged in a queue. All the processors connect to DRAM partitions via interconnection network.
+A processor array consists of many processing cores. A unified GPU processor array has a typical organized structure of multi-threaded multi-processors. For the execution of each thread, a multiprocessor is involved, and in each GPUs multi-processor, also known as Streaming Multiprocessors (SM), there are numerous Streaming processors, arranged in a queue. All the processors connect to DRAM partitions via the interconnection network.
 ### Multi-Threading
-As discussed earlier, GPU is optimized for high throughput and latency hiding. High scale multithreading shrinks the latency of memory loads from DRAM. While a thread is at stall because of a load or fetch instruction to complete, the processor can execute another thread. Also, because of high scale multithreading, GPU supports fine-grained parallel graphics shader programming models and fine-grained parallel computer programming models.
+As discussed earlier, GPU is optimized for high throughput and latency hiding. High scale multithreading shrinks the latency of memory loads from DRAM. While a thread is at stall because of a load or fetches instruction to complete, the processor can execute another thread. Also, because of high scale multithreading, GPU supports fine-grained parallel graphics shader programming models and fine-grained parallel computer programming models.
```

Comment:
```suggestion
As discussed earlier, GPU is optimized for high throughput and latency hiding. High scale multi-threading shrinks the latency of memory loads from DRAM. While a thread is at stall because of a load or is fetching instructions to complete, the processor can execute another thread. Also, because of high scale multi-threading, GPU supports fine-grained parallel graphics shader programming models and fine-grained parallel computer programming models.
```
Change any other instances of multithreading to multi-threading.